code/datums/ai/basic_mobs/basic_ai_behaviors/targeting.dm

```diff
@@ -4,11 +4,15 @@
 	var/vision_range = 9
 	/// Blackboard key for aggro range, uses vision range if not specified
 	var/aggro_range_key = BB_AGGRO_RANGE
-	/// Static typecache list of potentially dangerous objs
-	var/static/list/hostile_machines = typecacheof(list(/obj/machinery/porta_turret, /obj/vehicle/sealed/mecha))
+	/// Static typecache list of things we are interested in
+	var/static/list/interesting_atoms = typecacheof(list(/mob, /obj/machinery/porta_turret, /obj/vehicle/sealed/mecha))
+
+/datum/ai_behavior/find_potential_targets/get_cooldown(datum/ai_controller/cooldown_for)
+	if(cooldown_for.blackboard[BB_FIND_TARGETS_FIELD(type)])
+		return 60 SECONDS
```

Comment:
feels arbitary?  players might think its broken with such a long delay

Reply:
In theory the delay doesn't really matter since any new mobs will trigger the field and get detected, I think? this was just a backup, tho I could be misremembering the logic.